### PR TITLE
Rate override

### PR DESCRIPTION
**Pull request by interlock**
Feature to add custom rates for any user - project - task combination.
- Admin "Rates" editing area for creating, updating and deleting these rates
- New Time Entries prefer to use a Rate if it exists

How's it look @ggrochow ?

### CODE REVIEW

File: lib/tasks/statement.rake [hunk 1]
```diff
@@ -23,7 +24,7 @@ namespace :statement do
   end
 
   desc "Lock all statements with a post_date of either today / the date passed into DATE, ex DATE=2016-01-15"
-  task daily_lock: :environment do
+  task :daily_lock, :environment do
```
Comment (ggrochow) [inline lib/tasks/statement.rake]: Was this not properly loading the environment?
does it fix the logger issue

**Comment by ggrochow** [comment]
I would also look into the update time entry interactor

File: lib/tasks/statement.rake (hunk 1 above)
Comment (interlock) [inline lib/tasks/statement.rake]: I actually could not get rspec to work right, so I reverted the puts to logger.info for now :-(

**Comment by interlock** [comment]
@ggrochow I'll take a look, but generally I haven't noticed edits updating rate. Perhaps they should anyways.

**Comment by interlock** [comment]
@ggrochow Updated the other interactor as mentioned. Added some cross linking and spec tests.

**Pull request merged**